Tests/Microsoft.Azure.Mobile.Test.Windows/Channel/ChannelTest.cs

```diff
@@ -154,6 +155,102 @@ public void ChannelInvokesFailedToSendLogEvent()
             Assert.IsTrue(FailedToSendLogOccurred(MaxLogsPerBatch));
         }
 
+        /// <summary>
+        /// Validate that links are same on an error and a log
+        /// </summary>
+        [TestMethod]
+        public void FailedToSendLogEventArgsAreSame()
+        {
+            var ex = new Exception();
+            var log = new TestLog();
+            var failedEventLogArgs = new FailedToSendLogEventArgs(log, ex);
+            Assert.AreSame(log, failedEventLogArgs.Log);
+            Assert.AreSame(ex, failedEventLogArgs.Exception);
+        }
+
+        /// <summary>
+        /// Validate that channel'll send log after enabling
+        /// </summary>
+        [TestMethod]
+        public void ChannelInvokesSendingLogEventAfterEnabling()
+        {
+            _channel.Shutdown();
+            for (int i = 0; i < MaxLogsPerBatch; ++i)
+            {
+                _channel.Enqueue(new TestLog());
+            }
+            _channel.SetEnabled(true);
+
+            Assert.IsTrue(SendingLogOccurred(MaxLogsPerBatch));
+        }
+
+        /// <summary>
+        /// Validate that FailedToSendLog calls when channel is disabled
+        /// </summary>
+        [TestMethod]
+        public void ChannelInvokesFailedToSendLogEventAfterEnabling()
```

Comment:
Should the name of this be after disabling? Also, when I run this test on its own it passes, but when I select "run all" tests in visual studio, this one fails.